logging/Notes_on_using_TLS.md

```diff
@@ -2,7 +2,21 @@
 
 ## Overview
 
-TLS is required for communication between the Elasticsearch nodes and for accessing Elasticsearch from other components. It is also a best practice to use TLS (by using HTTPS) when accessing Kibana from a browser.
+TLS enablement for SAS Viya logging is divided into two parts:
+
+- **TLS for in-cluster communications**, which is between logging components within
+the cluster (between the Elasticsearch nodes and between Elasticsearch and other logging components). TLS must be enabled on these connections.
+
+- **TLS for communications into the cluster**, which is between a user's 
+browser and Kibana. TLS is optional on these connections, although enabling
+TLS is a best practice.
+
+The information in this document explains only how to configure TLS 
+for in-cluster communications, either automatically (using cert-manager) 
+or manually.
+
+For information about configuring TLS for communications into the cluster, see 
+the [TLS logging sample](../samples/tls/loggingREADME.md).
 
```

Comment:
I wonder whether it might be more clear to move these two paragraphs to immediately follow the category of TLS enablement they address rather than introduce the two categories and then follow it with two more paragraphs that each address one of the categories.
So, organize things like:

- TLS for in-cluster communications
- This document explains how to...
- TLS for communications into the cluster
- For information about configuring TLS for communication into the cluster, see...

As it is currently organized, the reader has to think about A then B and then back to A and then back to B.